crates/teloxide/src/dispatching/dialogue.rs

```diff
@@ -141,13 +141,13 @@ where
     /// Constructs a new dialogue with `storage` (where dialogues are stored)
     /// and `chat_id` of a current dialogue.
     #[must_use]
-    pub fn new(storage: Arc<S>, chat_id: ChatId) -> Self {
+    pub const fn new(storage: Arc<S>, chat_id: ChatId) -> Self {
         Self { storage, chat_id, _phantom: PhantomData }
     }
 
     /// Returns a chat ID associated with this dialogue.
     #[must_use]
-    pub fn chat_id(&self) -> ChatId {
+    pub const fn chat_id(&self) -> ChatId {
```

Comment:
This is pretty much pointless -- you can't allocate `Arc` in constant context, so while you can theoretically call these functions, you can't create the arguments.